Makefile

```diff
@@ -97,15 +77,19 @@ lint: $(GOLANGCI_LINT) deps shellcheck jsonnet-fmt
 	$(GOLANGCI_LINT) run -v --enable-all -c .golangci.yml
 
 .PHONY: test
-test: build test-unit test-integration
+test: build test-unit
 
 .PHONY: test-unit
 test-unit:
-	CGO_ENABLED=1 GO111MODULE=on go test -v -race -short ./...
+	CGO_ENABLED=1 GO111MODULE=on go test -v -race -short $(shell go list ./... | grep -v api/test/e2e)
+
+.PHONY: test-e2e
+test-e2e: container
+	CGO_ENABLED=1 GO111MODULE=on go test -v -race -short -tags integration ./test/e2e
 
-.PHONY: test-integration
-test-integration: build integration-test-dependencies generate-cert
-	THANOS=$(THANOS) UP=$(UP) DEX=$(DEX) LOKI=$(LOKI) WEBSOCAT=$(WEBSOCAT) OPA=$(OPA) GUBERNATOR=$(GUBERNATOR) ./test/integration.sh
+.PHONY: test-interactive
+test-interactive: container
```

Comment:
What a great idea - so many time I just put `time.sleep(10 * time.Minute)` into my tests :smile:

Reply:
Blatantly ripped of from Thanos interactive test :stuck_out_tongue_winking_eye: